### DISCUSSION TITLE
How to disable done.txt and report.txt files

### DISCUSSION THREAD

**Discussion by fritzrehde** [todotxt/todo.txt-cli#414] [Q&A] [answered]
I don't care much about the tasks that I have completed, and for the sake of simplicity, I would like to only have the todo.txt file be generated (I don't want done.txt or report.txt). Is there a setting for this?

**Comment by inkarkat** [comment, marked as answer]
All file locations can be configured in the `todo.cfg` config file (found in various alternative locations in your home directory - you should have created one from the template when you started using todo.txt-cli). The relevant settings are:

```
export DONE_FILE="/dev/null"
export REPORT_FILE="/dev/null"
```

With `/dev/null`, you will never see those files, and anything you archive will be gone immediately. An alternative would be a file under e.g. `/tmp` to have a chance at getting back the contents, or maybe a file location where your user has no access (like `/opt/todo-archive-disallowed`) so that you'll get an error message if you accidentally trigger the archiving.
